scripts/modify-layers-pdok-ogcapi.py

```diff
@@ -0,0 +1,172 @@
+import json
+import os
+import shutil
+import requests
+import sys
+
+# python script to modify layers-pdok for the plugin.
+# provide a single parameter to determine behaviour of the script, either:
+# ogcapi `python3 ./scripts/modify-layers-pdok-ogcapi.py ogcapi`: Adds ogcapi test records to layers-pdok.json and creates a copy
+# original `python3 ./scripts/modify-layers-pdok-ogcapi.py original`: Replaces layers-pdok.json with the original copy and removes the copy
+
+
+def extend_layer_pdok_ogcapi(urls_oaf = [], urls_oat = []):
+        layers_pdok = []
+        layers_pdok = retrieve_layers_from_oat_endpoint(urls_oat)
+        layers_pdok.extend(retrieve_layers_from_oaf_endpoint(urls_oaf))
+        return layers_pdok
+    
+def retrieve_layers_from_oat_endpoint(urls = []):
+    oat_layers = []
+    for url in urls:
+        url_info = requests.get(url).json()
+        dataset_title = url_info.get('title', url.split('/')[-1])
+        tiles_info = requests.get(url + "/tiles").json()
+        # For the two test datasets, we hardcode some information for now which can not yet be retrieved from the endpoint
+        if 'bag' in url:
+            dataset_md_id = "aa3b5e6e-7baa-40c0-8972-3353e927ec2f"
```

Comment:
Het is goed om dit soort magische constantes in een `CONSTANTE` bovenaan het script te plaatsen. Ook bij dit soort scripts.